doc/REST-interface.md

```diff
@@ -5,6 +5,8 @@ The REST API can be enabled with the `-rest` option.
 
 Supported API
 -------------
+
+####1. Transactions
```

Comment:
I'm not sure adding a number here is a good idea - this makes editing harder, as sections have to be manually renumbered. As markdown has no way to auto-number sections (IIRC) it's better to just leave it.